apigw/src/internal/routes/auth-status.ts

```diff
@@ -19,20 +21,39 @@ export default toRequestHandler(async (req, res) => {
         res.status(200).json({
           loggedIn: true,
           user: { id, name, unitId },
-          roles: user.roles
+          roles:
+            user.roles ??
+            concat(user.globalRoles ?? [], user.allScopedRoles ?? [])
```

Comment:
Why not add `globalRoles` to the response as a separate property? Or is the idea to keep using this old `roles` property in the frontend?

Reply:
I didn't want to include frontend changes here at this point, so it still uses the old property.

But I think I'll add the two new properties to the response, so we can already return the data in the correct format even though the frontend doesn't yet use it -> less chance for breakage during upgrade